tests/sqlparser_common.rs

```diff
@@ -14464,6 +14468,16 @@ fn test_case_statement_span() {
     );
 }
 
+#[test]
```

Comment:
Could we move the test case to the [spans.rs](https://github.com/apache/datafusion-sqlparser-rs/blob/6cf531d4951545708a82b208bed8ea596f91a186/src/ast/spans.rs#L2333) tests?

Reply:
Absolutely! I moved it and aligned its body with the approach taken in the rest of the tests in `spans.rs`. Thank you for the review 🥰!